packages/good-design/src/core/buttons/ClaimButton.tsx

```diff
@@ -67,8 +67,8 @@ const ClaimButton = ({
           </>
         ),
         footer: (
-          <View justifyContent="space-between" width="full" flexDirection="row">
-            <ActionButton color="white" text={"Verify Uniqueness"} onPress={verify} bg="main" />
+          <View justifyContent="center" width="full" flexDirection="row">
+            <ArrowButton text={"VERIFY I'M HUMAN"} onPress={verify} textInteraction={{ hover: { color: "white" } }} />
```

Comment:
to support i18n we need to have an option to pass custom title.
let this text be default prop +option to pass any other title from the protocol side

Reply:
the same for the other texts I think (need to creat separate issue, discuss it with @sirpy and @patpedrosa )

Reply:
https://github.com/GoodDollar/GoodWeb3-Mono/issues/57

Created, I do feel like this is not necessarily in the scope of MVP next release
Lets just keep it hardcoded momentarily until a proper solution has been discussed?